integration-cli/docker_cli_daemon_test.go

```diff
@@ -447,6 +450,249 @@ func (s *DockerDaemonSuite) TestDaemonExitOnFailure(c *check.C) {
 	}
 }
 
+func (s *DockerDaemonSuite) TestDaemonBridgeExternal(c *check.C) {
+	d := s.d
+	err := d.Start("--bridge", "nosuchbridge")
+	c.Assert(err, check.NotNil, check.Commentf("--bridge option with an invalid bridge should cause the daemon to fail"))
+	defer d.Restart()
```

Comment:
The current IT infra doesn't have a dedicated namespace for the Docker daemon that is Started. Hence the new daemon shares the resources with the Daemon that is already running.
Any intrusive changes made to the IPTables, Bridge Configuration etc. might affect other tests that follows.
Hence by issuing a `Restart()` as a defered action (without any parameter), we are guaranteed to have the shared resources reset to defaults. This makes even intrusive tests like these act nice with all other tests.

A better approach would be to run the DaemonSuite in a different Namespace. But we can address that change later.